Buildings/Fluid/Humidifiers/Data/Generic.mo

```diff
@@ -0,0 +1,38 @@
+within Buildings.Fluid.Humidifiers.Data;
```

Comment:
It would be better if we could have a set of example data in the package. For example, see `Buildings.Fluid.HeatPumps.Data.EquationFitReversible`

Reply:
@JayHuLBL We actually have an example in the Validation package (Humidifiers.Validation.Data). Should we move it this package?

Reply:
@JayHuLBL Is the data from the EnergyPlus reference?

Reply:
Yes. The parameter values came from the EPlus model that was used for the validation process

Reply:
Please add it to the `Data` package